HAB5/hab5.tex

```diff
@@ -0,0 +1,404 @@
+Potential payloads also include ``training'' for payloads utilizing machine learning. 
+All aspects SPEX greatly benefits from having.
+
+Another aspect to SPEX is the learning in the process, with documentation specified as a deliverable. 
+Also, by incorporating testing and training processes into the design and execution process, SPEX can iterate on more advanced software tools. 
+Specific focus on algorithmic improvisation may also draw in more talent, which in turn advances the scope of possible missions.
+
+Computer vision is field skirting the line between nascent and burgeoning. 
+As image processing tools advance, ever more useful information can be garnered from the same physical equipment. 
+Progress here automatically increases SPEX's potential to make news, capture imaginations and talent, and grow, for we will be at the relative forefront of technology---unlike projects aiming at aero-braking/recovery, novel communication or astronomical tracking, which are likely to lag behind by decades. 
+This advantage is both due to us maximizing the knowledge gleamed from cheap and readily available hardware, as well as the abundance of open source resources for computer vision. 
+While the entry requirements are non-zero, the acquisition of talent utilizing OpenCV or useful Python libraries broadens SPEX's options for other ventures, such as IREC's SDL Payload Challenge. 
+
+Further, this project offers great flexibility in terms of team composition, once beyond a requisite minimum.
+The scope for inclusion of mechanically ``solved'' or immensely complicated designs is immense, while the electrical or computer science requirement becomes static after surpassing a threadbare. 
+An electrically gifted team may choose to consider much more advanced imaging equipment or image processing using other factors to control, requiring only a moderate CS investment. 
+Similarly, the potential for much more advanced algorithmic excursions on maintained or minimally modified hardware is substantial. 
+
+As RIT looks to expand its research endeavors, especially in fields such as Imaging Science where it possesses an advantage, developing a long and mutually beneficial relationship with that department is also a goal unto itself. 
+By acting as an experimentalist's platform, SPEX (HAB) has the unique opportunity to be a student body involved in the synthesis, publication and presentation of materials. 
+Be it in academic journals or otherwise, our unique selling point can enable increasing reach and developing leads and future deliverables allowing for assistance and funding from sustainable sources.
+
+\section{Implementation}
+\label{implementation}
+\subsection{Deliverables}
+\label{deliverables}
+\subsubsection{Documentation}
+\label{deliverables-documentation}
```

Comment:
Just a note - one of the things I've always wanted SPEX to do would be a 'how-to-launch-a-hab' technical report with everything that be beginner would need to know as to launch a hab. Just an idea. Could be a cool deliverable.

Reply:
👍

Reply:
We already have this. There is an extremely thorough document on the google drive.